dss-pades-pdfbox/src/test/java/eu/europa/esig/dss/pades/PdfBoxUtilsTest.java

```diff
@@ -60,42 +63,42 @@ void init() {
 
 	@Test
 	void generateScreenshotTest() {
-		DSSDocument screenshot = PdfBoxUtils.generateScreenshot(sampleDocument, 1);
+		DSSDocument screenshot = PdfBoxUtils.generateScreenshot(sampleDocument, 1, MemoryUsageSetting.setupMainMemoryOnly());
```

Comment:
do you want I create both version of each method?
I won't keep only the old PdfBoxUtils methods since that will always load into memory and it is not the deal. Am I correct?

Reply:
Yes, it would be desirable to have overloaded methods, so simple as well as advanced configuration is possible.
If it becomes too many of variables, we may also think about creation of a separate class for screenshot generation, for instance using a builder or a factory design patterns.